app/views/api/v1/api_docs/oauth2_redirect.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html lang="en-US">
```

Comment:
Is this pulled from a gem or example somewhere, or is this something you coded?  If the former, would be nice to link the source, if the latter, a brief comment at the top of why this file was necessary would be nice.  I know it's to help with Swagger oauth, but I'm curious why we don't get this behavior for free from the gem.   So just something like "foo gem doesn't support Oauth foo, so we have to provide an explicit bar"

Reply:
We can't use the gem because there's a bug with Sprockets 4 & [`swagger_ui_engine`](https://github.com/zuzannast/swagger_ui_engine). The two are incompatible, so we had to rip out the gem for rendering the Swagger frontend.  We're using the standalone distribution that is sourced directly from the official [Github](https://github.com/swagger-api/swagger-ui/blob/master/dist/index.html) repo, with some small customizations to make it work with SCP (using a CDN for the JS/CSS, and the configuration to read from our `swagger-blocks` yaml).  I'll put all that in a comment.

Reply:
2b00741